src/proof/lfsc/lfsc_node_converter.h

```diff
@@ -0,0 +1,113 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Andrew Reynolds
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2021 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * Implementation of LFSC node conversion
+ */
+#include "cvc5_private.h"
+
+#ifndef CVC4__PROOF__LFSC__LFSC_NODE_CONVERTER_H
+#define CVC4__PROOF__LFSC__LFSC_NODE_CONVERTER_H
+
+#include <iostream>
+#include <map>
+
+#include "expr/node.h"
+#include "expr/node_converter.h"
+#include "expr/type_node.h"
+
+namespace cvc5 {
+namespace proof {
+
+class LfscNodeConverter : public NodeConverter
+{
+ public:
+  LfscNodeConverter();
+  ~LfscNodeConverter() {}
+  /** convert to internal */
+  Node postConvert(Node n) override;
+  /** convert to internal */
+  TypeNode postConvertType(TypeNode tn) override;
+  /**
+   * Get the null terminator for kind k and type tn. The type tn can be
```

Comment:
Maybe clarify with an example of null terminators?

Reply:
Done